.travis.yml

```diff
@@ -5,7 +5,7 @@ node_js:
 
 env:
   - COMMAND=build
-  - COMMAND=test
+  - COMMAND=test-ci
```

Comment:
I think for the sake of clarity test-coverage would be better name than test-ci :]